### ISSUE TITLE
Lane changer

### ISSUE THREAD

**Issue by thaddeus** [chrisblatchley/sumo-cuda#10]
Is this entirely necessary for the minimized scope of our project? Might be simpler to use some algorithm to check for lane changing, probably would allow better parallel speeds this way.

**Comment by chrisblatchley** [comment]
Yeah, i was just thinking about that. I'm thinking LaneChanger is not necessary
